catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -292,6 +292,11 @@
             <groupId>org.apache.zookeeper</groupId>
             <artifactId>zookeeper</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Given the substantial changes to the CatalogFrameworkImplTest, should the COVEREDRATIOs of this file be updated?